### ISSUE TITLE
Testing framework

### ISSUE THREAD

**Issue by AleksueiR** [HaNdTriX/generator-chrome-extension-kickstart#14]
Is it possible to add a testing framework to the generator? Thank you.

**Comment by HaNdTriX** [comment]
Yes it is. I haven't added it to the boilerplate yet, because there are so many different ways to do your tests.

Just create a `test.js` file in the tasks directory (`./tasks/test.js`).
This will get automatically loaded via gulp.

    import gulp from 'gulp';
    import livereload from 'gulp-livereload';
    import args from './lib/args';

    gulp.task('test', () => {
      return gulp.src('./test/index.js')
        ...
        .pipe(gulpif(args.watch, livereload()));
    });

(Replace `...` with your own config)

More Info: https://github.com/sindresorhus/gulp-mocha
